test/e2e/authorizer/authorizer_test.go

```diff
@@ -151,32 +104,43 @@ func TestAuthorizer(t *testing.T) {
 	require.Equal(t, len(f.Servers), 1, "incorrect number of servers")
 
 	server := f.Servers["main"]
+
 	kcpCfg, err := server.Config()
 	require.NoError(t, err)
-	clusterName, err := framework.DetectClusterName(kcpCfg, ctx, "workspaces.tenancy.kcp.dev")
+	kubeClusterClient, err := kubernetes.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kubeClients, err := kubernetes.NewClusterForConfig(kcpCfg)
+	kcpClusterClient, err := kcp.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kcpClients, err := kcp.NewClusterForConfig(kcpCfg)
+	dynamicClusterClient, err := dynamic.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kcpClient := kcpClients.Cluster(clusterName)
-	require.NoError(t, err)
-	dynamicClient, err := dynamic.NewForConfig(kcpCfg)
+
+	orgClusterName := "admin"
+	_, org, err := helper.ParseLogicalClusterName(orgClusterName)
```

Comment:
What does this function call accomplish?

Reply:
from what i see just validation

Reply:
for now it does nothing. When orgs become a thing in #508, this will make more sense.